app/assets/javascripts/administration.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function(){
+    $("ul#tabs li").click(function(e){
+        e.preventDefault();
```

Comment:
You mean the prevent default? Yeah, it was added when I first was trying to debug why it wasn't working.